buildbot/configure.py

```diff
@@ -23,6 +23,14 @@ def do_configure(args):
     libclc_amd_target_names = ';amdgcn--;amdgcn--amdhsa'
     libclc_nvidia_target_names = ';nvptx64--;nvptx64--nvidiacl'
 
+    sycl_enable_fusion = "OFF"
+    if not args.disable_fusion:
+        if platform.system() == 'Windows':
+            print("Warning: Kernel fusion not yet supported on Windows, will disable")
```

Comment:
This duplicates the logic in CMake files. I suggest we remove this code from the python script and leave in CMake files only.